src/shared/components/annotation/genomeNexus/Sift.tsx

```diff
@@ -82,7 +84,7 @@ export default class Sift extends React.Component<ISiftProps, {}> {
                         <td>
                             <span
                                 className={
-                                    tooltipStyles[
+                                    (tooltipStyles as any)[
                                         `sift-${this.props.siftPrediction}`
```

Comment:
instead you could try ```tooltipStyles[`sift-$....` as keyof tooltipStyles`]```. Not sure if it would work but it might

Reply:
hmm, I'll give it a try

Reply:
this doesn't seem to work, leaving it `as any` for now